packages/office-addin-manifest/test/test.ts

```diff
@@ -623,27 +624,56 @@ describe("Unit Tests", function() {
       });
     });
   });
+  describe("export.ts", function() {
+    describe("exportMetadataPackage()", function() {
+      it("export manifest", async function() {
+        this.timeout(6000);
+        const testFolder = path.resolve("./testExecution");
+        const manifestPath = path.normalize("test/manifests/manifest.json");
+        const assetsPath = path.normalize("test/assets");
+        const outputPath = path.normalize(`${testFolder}/testPackage.zip`);
+        const outputFile = await exportMetadataPackage(outputPath, manifestPath, assetsPath);
+
+        assert.strictEqual(outputFile, outputPath, "Output path should match the argument");
+        assert.strictEqual(fs.existsSync(outputFile), true, "Output file should exist");
+        
+        // Cleanup
+        fs.rmSync(testFolder, { recursive: true });
```

Comment:
Can we use the `deleteFolder` function?

Reply:
Was trying that first, but for some reason using a method call creates some sort of ordering problem with the calls such that the delete ends up trying to delete the folder with the file still in it (and I didn't want to change the delete function to use this simpler call because I wasn't sure the side effect on the other usage since it's using specific operations within the function).